tooling/project-template/project-template.gradle

```diff
@@ -0,0 +1,112 @@
+import org.apache.tools.ant.filters.ReplaceTokens
+
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
+ */
+plugins {
+    id 'base'
+
+    // for publishing snapshots
+    id 'maven-publish'
+    id 'org.hibernate.build.maven-repo-auth'
+
+    // publishing to BinTray
+    id "com.jfrog.bintray"
+    id "nu.studer.credentials" version "2.1"
+}
+
+// NOTE : Had trouble using the `distribution` plugin, so manually using Jar/Tar tasks
+
+version = '0.1'
```

Comment:
So this is your problem here.  You are saying that the version for this module different from the larger ORM build

Reply:
thnx!